CHANGELOG.md

```diff
@@ -6,13 +6,17 @@ All notable changes to Wallaroo will be documented in this file.
 
 ### Fixed
 
+- Argument parsers in machida no longer require a program name to be passed
 
 ### Added
 
 - Added support for Fedora 26/27, Amazon Linux 2, Oracle Linux, Ubuntu Artful, and Debian Jessie/Buster Linux distributions via Wallaroo Up
-- Checkpointing protocol for regular checkpointing of application state.
-- Support for rollback to most recent checkpoint on recovery.
-- Support for replicated recovery data.
+- Checkpointing protocol for regular checkpointing of application state
+- Support for rollback to most recent checkpoint on recovery
+- Support for replicated recovery data
+- Added preview release of the Python Connectors API
+- Added conncetor scripts for Kafka, Kinesis, RabbitMQ, Redis, S3, and UDP
```

Comment:
typo: `conncetor` should be `connector`

Reply:
This typo is stuck in my muscle memory somehow.